packages/conf-gegl/conf-gegl.1/opam

```diff
@@ -0,0 +1,19 @@
+opam-version: "2.0"
+maintainer: "https://github.com/fccm2"
+homepage: "https://gegl.org/"
+authors: "Øyvind Kolås"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+license: "LGPL-3.0-or-later, and GPL-3.0-or-later"
+synopsis: "Virtual package relying on a gegl lib system installation"
+description: """
+This package checks if the gegl lib is installed in the system.
+"""
+depends: [ "conf-pkg-config" ]
+depexts: [
+  ["libgegl-dev"] {os-family = "debian"}
+  ["libgegl-dev"] {os-family = "ubuntu"}
```

Comment:
Looks like we could also add these?

- https://formulae.brew.sh/formula/gegl
- https://pkgs.alpinelinux.org/package/edge/community/x86_64/gegl-dev
- https://packages.fedoraproject.org/pkgs/gegl04/gegl04/
- https://build.opensuse.org/package/show/graphics/gegl
- https://archlinux.org/packages/extra/x86_64/gegl/

WDYT?